.github/workflows/pypi-release.yml

```diff
@@ -0,0 +1,25 @@
+name: PyPI release
+on: [push]
+
+jobs:
+  pypi_release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.x
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.x
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade setuptools wheel twine
+    - name: Build
+      run: |
+        python setup.py sdist bdist_wheel
+        twine check dist/*
+    - name: Publish package
+      if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
+      uses: pypa/gh-action-pypi-publish@v1.1.0
```

Comment:
Minor: why not the latest version which is 1.4.2?
There're interesting features along the way like `twine check` running automatically (in 1.2.0), verbose settings for upload debugging (1.4.0) and few bug fixes

Reply:
No specific reason for that, just copy-pasted the same version We're using in `kivy-ios` :)
Updated to `v1.4.2`.